src/codec_svt.c

```diff
@@ -114,7 +114,11 @@ static avifResult svtCodecEncodeImage(avifCodec * codec,
         // See https://gitlab.com/AOMediaCodec/SVT-AV1/-/issues/1697.
         memset(svt_config, 0, sizeof(EbSvtAv1EncConfiguration));
 
+#if SVT_AV1_CHECK_VERSION(3, 0, 0)
+        res = svt_av1_enc_init_handle(&codec->internal->svt_encoder, svt_config);
+#else
         res = svt_av1_enc_init_handle(&codec->internal->svt_encoder, NULL, svt_config);
+#endif
```

Comment:
There is another change in SVT-AV1 v3.0. The type of `svt_config->force_key_frames` changed from `Bool` (defined in  EbSvtAv1.h) to `bool` (defined in <stdbool.h>).

So we should add `#include <stdbool.h>` and change the line

```
        svt_config->force_key_frames = TRUE;
```

to

```
        svt_config->force_key_frames = true;
```

Reply:
Good catch, done

Reply:
Disclosure: This suggestion is not mine. It comes from [libavif_api_job_fix.patch](https://gitlab.com/AOMediaCodec/SVT-AV1/-/blob/master/.gitlab/workflows/linux/libavif_api_job_fix.patch?ref_type=heads) :-)